README.md

```diff
@@ -25,597 +21,235 @@
--->
-
 </div>
 
 ---
 
-__Note:__ Flash is currently being tested on real-world use cases and is in active development. Please [open an issue](https://github.com/PyTorchLightning/lightning-flash/issues/new/choose) if you find anything that isn't working as expected.
-
----
-
-## News
-
-- Jul 12: Flash Task-a-thon community sprint with 25+ community members
-- Jul 1: [Lightning Flash 0.4](https://devblog.pytorchlightning.ai/lightning-flash-0-4-flash-serve-fiftyone-multi-label-text-classification-and-jit-support-97428276c06f)
-- Jun 22: [Ushering in the New Age of Video Understanding with PyTorch](https://medium.com/pytorch/ushering-in-the-new-age-of-video-understanding-with-pytorch-1d85078e8015)
-- May 24: [Lightning Flash 0.3](https://devblog.pytorchlightning.ai/lightning-flash-0-3-new-tasks-visualization-tools-data-pipeline-and-flash-registry-api-1e236ba9530)
-- May 20: [Video Understanding with PyTorch](https://towardsdatascience.com/video-understanding-made-simple-with-pytorch-video-and-lightning-flash-c7d65583c37e)
-- Feb 2: [Read our launch blogpost](https://pytorch-lightning.medium.com/introducing-lightning-flash-the-fastest-way-to-get-started-with-deep-learning-202f196b3b98)
+<div align="center">
+  <b>
+    Flash enables you to easily configure and run complex AI recipes for over 15 tasks across 7 data domains
+  </b>
+</div>
 
----
+<div align="center">
+  <a href="https://lightning-flash.readthedocs.io/en/stable">
+    <img src="https://pl-flash-data.s3.amazonaws.com/assets/banner.gif">
+  </a>
+</div>
 
-## Installation
+## Getting stated
 
-Pip / conda
+From PyPI:
 
 ```bash
 pip install lightning-flash
 ```
 
-<details>
-  <summary>Other installations</summary>
-
-Pip from source
-
-```bash
-# with git
-pip install git+https://github.com/PytorchLightning/lightning-flash.git@master
-
-# OR from an archive
-pip install https://github.com/PyTorchLightning/lightning-flash/archive/master.zip
-```
-
-From source using `setuptools`
-``` bash
-# clone flash repository locally
-git clone https://github.com/PyTorchLightning/lightning-flash.git
-cd lightning-flash
-# install in editable mode
-pip install -e .
-```
-
-In case you want to use the extra packages from a specific domain (image, video, text, ...)
-```bash
-pip install "lightning-flash[image]"
-```
-See [Installation](https://lightning-flash.readthedocs.io/en/latest/installation.html) for more options.
-</details>
-
----
-
-## What is Flash
-Flash is a framework of tasks for fast prototyping, baselining, finetuning and solving business and scientific problems with deep learning. It is focused on:
-
-- Predictions
-- Finetuning
-- Task-based training
+See [our installation guide](https://lightning-flash.readthedocs.io/en/latest/installation.html) for more options.
 
-It is built for data scientists, machine learning practitioners, and applied researchers.
+## Flash in 4 steps
```

Comment:
I think that 3 steps would be better here, then have predictions / serving as additional stuff later on in the readme similar to augmentations

Reply:
Done !